internal/conformance/runconformance.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+cd "$(dirname $0)"
+
+BINDIR="../../.tmp/bin"
+mkdir -p $BINDIR
+GO="${GO:-go}"
+
+# These will get built using current HEAD of this connect-go repo
+# thanks to replace directive in go.mod. So by testing the reference
+# implementations (which are written with connect-go), we are effectively
+# testing changes in this repo.
+"$GO" build -o $BINDIR/connectconformance connectrpc.com/conformance/cmd/connectconformance
+"$GO" build -o $BINDIR/referenceclient connectrpc.com/conformance/cmd/referenceclient
+"$GO" build -o $BINDIR/referenceserver connectrpc.com/conformance/cmd/referenceserver
```

Comment:
Does `"$GO"` need to be quoted here? I don't think so, since we don't care about word splitting.

Reply:
🤷, if we need quotes around `"$0"` (which I agree that I should add), then we probably want it here, too, as a just in case and for the same reason. Like if someone did something like `GO="/mnt/c/Program Files/cygwin/bin/go"`.

I prefer to quote anything that is user-provided since we don't for sure whether it will have whitespace, and whitespace is totally allowed in file paths, so seemed fair to do this way.

Or were you thinking that someone might want to use whitespace to do a `go` command with add'l args, like `GO="go-wrapper --some arg -v"`?

Reply:
Went ahead and made the requested change, assuming you were thinking of a go command with args.